src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,104 @@
+
+<figure role="figure" aria-label="Add image descriptions (alt text) to images in documents, as in Microsoft Word's Alt Text option.">
+	<img alt="Word document has photo inserted and selected. In the right pane, the Alt Text option is open and showing an input field that reads 'White and yellow flowers'." src="/img/posts/2021-06-14-Everyday_Accessibility/Windows-AltText.png" />
+	<figcaption>Add image descriptions ("alt text") to images in documents, as in Microsoft Word's Alt Text option.</figcaption>
+</figure>
+
+**Use accessibility checkers:** Microsoft products (namely Word and PowerPoint) come with a “Check Accessibility” option. Use this to double-check you’ve addressed key accessibility needs.
+
+<figure role="figure" aria-label="Microsoft's Check Accessibility feature helps ensure your document is inclusive.">
+	<img alt="The Check Accessibility menu option is activated on a simple PowerPoint deck where one slide is missing a title. The Accessibility pane shows the 'Missing slide title' error and the mouse hover text for that slide also reads '[No Title]'." src="/img/posts/2021-06-14-Everyday_Accessibility/PPT_A11yChecker.png" />
+	<figcaption>Microsoft's "Check Accessibility" feature helps ensure your document is inclusive.</figcaption>
+</figure>
+
+## Audio and Video Content
+
+**Have clear audio:** Though all of our devices can record audio and video on-demand, sound quality can vary.  Be sure to speak as clearly as possible, preferably at a steady pace and reasonable volume, and remove or avoid background noise including echo, empty room effect, humming, or white noise.
+
+> Using a microphone (even a common corded headset) can help amplify voices and minimize distracting unwanted sounds.
+
+**Reduce background music:** Instrumental music can enhance content’s quality, but not when it reduces the ability to hear the main speaker. When using background music, keep it to a minimum (such as during an intro) or, if used throughout, ensure it is very low.
+
+> Consider putting your background music at the volume of a whisper.
+
+**Provide text with multimedia:** As with images, video and audio can be engaging but also inaccessible without a text equivalent for people who are deaf-blind, having sensory processing disorders, or have hearing impairments. Having media content written out as text is vital for inclusiveness. Text equivalents can include transcripts that type out spoken dialogue, or an article that accompanies a video (as in a video that supplements a news article).
+
+> Where there’s media, make sure there’s also text.
+
+**Ensure you have no flashing content:** Content that flashes on the screen more than 3 times per second is dangerous as it can trigger a seizure in some viewers. Ensure any rapidly moving content is below this threshold.
```

Comment:
That's what I was trying to say with "rapidly moving content" without repeating "flashing content" again. The guideline should read that you _can't_ do flashing content.
Maybe we can change the heading to be more direct? E.g., "Do not have any flashing content".
Then the paragraph can say: "Content that flashes on the screen more than 3 times per second is dangerous as it can trigger a seizure in some viewers. Use rapid movement sparingly in your videos and ensure nothing flashes."

Reply:
This one is my bad: I did not parse your intro text correctly. Looking at this again, I think we agree. I think whatever language you settle on should emphasize having no flashing content at all, but if you must, keep flashes brief, etc etc.